docs/install.md

```diff
@@ -1,51 +1,73 @@
+ * [Visual Studio Code](http://code.visualstudio.com/) `Text editor which includes support for github, editorconfig and much more`
 
 Install the Dota 2 Workshop Tools.
  * From within steam, right click on `Dota 2` and select `Properties` at the bottom
  * Go to the DLC tab of the window that pops up
  * Check the `INSTALL` box next to `Dota 2 Workshop Tools DLC`
 
-## Addon Tool
+## Setting evrything up
 Open the `node.js command prompt`. Not `Node.js` itself, but the command prompt.
 
-### Install Node Dependencies
-Run the command 
+### Folder definitions
+Now we will define 3 paths 
+ * `Dota2 installation folder` is something like `C:\Program Files (x86)\Steam\steamapps\common\dota 2 beta\` or at ant other drive or install location, but certainly ends with `dota 2 beta`
+ * `Dota2 addons folder` is folder named `addons` in `Intallation folder`, so it looks something like `\dota 2 beta\addons`
+ * `Open Angel arena folder` is a folder name `oaa` inside of `addons folder`, it contains all the files which are present in this github repo 
+
+**`Dota2 addons folder` and `Open Angel arena folder` may not exist for the first time, they will be created after `Prepare Addon Directory` step**
+
+Final structure sholud look like `C:\(some path to steam library)\Steam\steamapps\common\dota 2 beta\addons\oaa\`
+
+### Navigating with command prompt
+Use command `cd` and the undergoing path to navigate,(`cd` works relative to your current path), for example
+```
+dota 2 beta\addons> cd oaa
+```
+will go from `Addons` folder to `Open Angel Arena` folder, and
+```
+dota 2 beta\addons\oaa> cd ../
 ```
-npm install -g gulp-cli dota2-addon-manager
+will navigate you back a step to `Addons` folder, and
 ```
-This will install both `gulp`, used for building npc files, and `d2am`, used to manage the addons directory.
+C:\somerandompath> D:
+```
+will change your current active drive to D:
 
-### Prepare Addon Directory
-Next find your `dota 2 beta` folder. You must change the directory of the node.js command prompt to this folder, this can be done with the `cd` command. It should look something like this...
+### Install Node Dependencies
+The global install for some reason works very confusingly, so i recommend using local repo for gulp.
+Now we navigate into `Dota2 installation folder` folder, and first install dependencies
 ```
-cd C:\Program Files (x86)\Steam\steamapps\common\dota 2 beta\
+npm install dota2-addon-manager
 ```
-**Note**: The directory path may be different depending on how you installed Steam. If Steam is installed on another drive you'll need to change drives using the letter then a `:` symbol. For example, `E:`.
 
-To ensure the addons directory is set up, run
+### Download and Deploy Project
+To ensure the addons directory is set up, navigate to `Dota2 Installation` folder and run
 ```
 d2am list
 ```
-leave the command prompt open, we'll come back to it. Join us on iscordDay, the URL ends in WNFBB4d. You found the easter egg!
```

Comment:
please do not remove this, it's the only way people find the discord right now. Also this has nothing to do with gulp.